CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ project(onnx-mlir)
 option(ONNX_MLIR_BUILD_TESTS "Build ONNX-MLIR test executables. If OFF, just generate build targets." ON)
 option(ONNX_MLIR_ENABLE_WERROR "Enable warnings as errors." OFF)
 option(ONNX_MLIR_SUPPRESS_THIRD_PARTY_WARNINGS "Suppress warning in third_party code." ON)
+option(ONNX_MLIR_APPEND_VC_REV "Embed the version control system revision in onnx-mlir." ON)
```

Comment:
I see this is set to ON to match the default settings of LLVM_APPEND_VC_REV which is currently also ON. 
I would have prefered the default to be OFF, but I think is better to match the default of LLVM_APPEND_VC_REV.